x-pack/plugins/security_solution/server/lib/detection_engine/rules/prepackaged_rules/defense_evasion_adding_the_hidden_file_attribute_with_via_attribexe.json

```diff
@@ -61,5 +61,5 @@
   "timeline_title": "Comprehensive Process Timeline",
   "timestamp_override": "event.ingested",
   "type": "eql",
-  "version": 10
+  "version": 11
```

Comment:
Another unnecessary version bump? Or was it missed in the previous batch of 8.2 updates, or bug in the Kibana rule generator?